core/domain/param_domain.py

```diff
@@ -144,6 +144,11 @@ def customization_args(self):
         return self._customization_args
 
     def to_dict(self):
+        """Returns a dict representing this Param domain object.
+
+        Returns:
+            dict. A dict representation of the Param instance.
```

Comment:
Add datatypes for key and value pairs for dict also like-- dict(str, str). Please check it.

Reply:
@vibhor98 if i am not wrong ,actually it is given in constructor

Reply:
@vibhor98  is it necessary to give Add datatype for key and value pair for dict because i checked and found that in any file in `core.domain ` that contain  docstrings and having `to_string`  method don't have datatypes for key and value pairs for dict .

Reply:
It's ok then. @seanlip , please have a final review.